lcls-twincat-optics/lcls_twincat_optics_plc/POUs/Helpers/FB_MirrorTwoCoatingProtection.TcPOU

```diff
@@ -25,13 +25,19 @@ VAR_INPUT
     nUpperCoatingBitmask : DWORD := 0;
 
     nLowerCoatingBitMask : DWORD := 0;
+
+    bMirrorTempFaults : BOOL := FALSE;
 END_VAR
 VAR_OUTPUT
 END_VAR
 VAR_IN_OUT
     FFO : FB_HardwareFFOutput;
+
 END_VAR
 VAR
+    // Coating Enums for local coding, not a readback.
+    E_CoatingPos : (Upper:=0, Lower:=1);
```

Comment:
I'm glad this value assignment syntax exists for inline enums, I didn't know about it